test/suite/test_txn19.py

```diff
@@ -354,7 +354,7 @@ def test_corrupt_log(self):
 
 class test_txn19_meta(wttest.WiredTigerTestCase, suite_subprocess):
     base_config = 'log=(archive=false,enabled,file_max=100K),' + \
-                  'transaction_sync=(enabled,method=none)'
+                  'transaction_sync=(enabled,method=none),cache_size=1GB'
```

Comment:
I don't fully understand what's going on here, but with the defaults, one case (`-s 4`, aka test_txn19.test_txn19_meta.test_corrupt_meta(removal.WiredTigerHS.wt)`) was getting stuck.  I made the change to both tests to keep them in sync.

Reply:
I'm not too concerned by that - this test isn't about stressing the cache, so I don't think we need to dig too deep.